torch/csrc/jit/passes/batch_mm.cpp

```diff
@@ -72,9 +72,7 @@ static constexpr size_t min_fusion_size = 2;
 
 static std::array<int64_t, 2> as_array(at::IntList sizes) {
   JIT_ASSERT(sizes.size() == 2);
-  std::array<int64_t, 2> arr;
-  arr[0] = sizes[0];
-  arr[1] = sizes[1];
+  std::array<int64_t, 2> arr = {sizes[0], sizes[1]};
```

Comment:
What was wrong here?

Reply:
It's the same issue as for all other garbage initialization cases in this PR

Reply:
But this initializes both elements, so what's the issue?